docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,149 @@
+
+The `prefect auth login` and `prefect auth logout` commands now use API keys. Previously, logging out with an API token would just reset your tenant and access token, but leave the API token on disk. Now, we retain that behavior the first time the command is called, but if you call it a second time, we delete the API token. This allows users to easily remove their API token in favor of an API key.
+
+Agents will use an available API key as detected by the Client, but will no longer detect tokens. If you have logged in locally, you may start any kind of agent and the agent will use your API key to query for flow runs, then pass the API key to the flow run for execution as well.
+
+When using the non-default tenant associated with an API key, set the tenant on the flow run job or the flow run will instantiate a client with the wrong tenant ID to report to Cloud. If the default tenant or a token is being used, this environment variable is just left blank.
+
+```bash
+# install agent, specifying API key and tenant
+$ prefect agent [agent-type] install --key [api-key] --tenant-id [tenant-id]
+
+# start agent, specifying API key and tenant
+$ prefect agent [agent-type] start --key [api-key] --tenant-id [tenant-id]
+```
+
+## Support for environments
+
+Flow environments, deprecated since Prefect 0.14.0, have been removed completely. Use `RunConfig` objects to define where and how a flow run should be executed as described in [Run Configuration](/orchestration/flow_config/run_configs.md).
+
+If you do not have `flow.environment` configured explicitly on your flow, there is no impact on your flows and no migration steps are needed.
+
+If you still have environment configurations, the topic [Upgrading Environments to RunConfig](/orchestration/faq/upgrade_environments.md) provides detailed instructions for migrating to `RunConfig`.
+
+## Registering and running flows with the CLI
+
+The Prefect CLI commands to register and run flows have been revised, changing the syntax and adding new functionality.
+
+### Registering flows
+
+The `prefect register` command replace the `prefect register flow` command to [Register a flow](/orchestration/getting-started/registering-and-running-a-flow.html#register-a-flow) with the CLI. 
+
+- Allows registering multiple flows in a single call. Flows can be specified by `--path` (path to a file or directory containing flows) or by `--module` (an importable Python module containing flows). Both options can be specified multiple times in a single call for more flexibility.
+- A `--name` flag can be used to only register flows with a specific name (or names). If unspecified, all flows found are registered.
+- By default, a flow will only be re-registered if it is *structurally different* than the existing version. This means that small edits to the source of tasks won't require re-registration, and the CLI will automatically detect this to avoid needlessly bumping the version. This can be disabled by passing in `--force`.
```

Comment:
This is true but not in all cases. I don't know how far in detail we want to go but this is specific to script based storages like Github. If you pickle (default behavior) like Local storage, you do need to register to take effect because you need the file in storage to change